package.json

```diff
@@ -55,7 +55,7 @@
   "devDependencies": {
     "@actions/core": "^1.10.1",
     "@arethetypeswrong/cli": "^0.15.3",
-    "@changesets/cli": "^2.26.2",
+    "@changesets/cli": "^2.27.8",
```

Comment:
@guustgoossens can you revert this change please?

Reply:
@alexcarpenter done!

Reply:
You'll need to run an `npm i` to remove the changes in the package-lock file as well.

Reply:
@alexcarpenter just did that, thanks for the patience.

Reply:
@guustgoossens no problem at all! Still seeing changes to the package-lock.json file. Can you revert those changes please?